bin/varnishtest/tests/c00103.vtc

```diff
@@ -0,0 +1,32 @@
+varnishtest "REGEX expressions in VCL"
+
+varnish v1 -vcl {
+	import debug;
+	backend be none;
+	sub vcl_recv {
+		# NB: the REGEX expression below is needlessly complicated
+		# on purpose to ensure we don't treat REGEX expressions
+		# differently.
+		if (req.url ~ (debug.just_return_regex(
+		    debug.just_return_regex("hello")))) {
```

Comment:
what happens when a dynamic string is used as a REGEX argument?

Reply:
With this, only CSTR tokens can be used, so what happens is a compile-time error.

Reply:
I was trying to imply that we should have a test ;)

Reply:
thank you, our comments have crossed.